war/src/main/js/add-item.js

```diff
@@ -116,6 +116,18 @@ $.when(getItems()).done(function(data) {
       }
     }
 
+    function focusButton(status) {
+      var btn = $('form .footer .btn-decorator button[type=submit]');
```

Comment:
I think add a `getFormSubmitButton` function that hides the jquery selector and can be reused here and in the `enableButton` function.